lib/Fmt_ast.ml

```diff
@@ -4393,17 +4397,30 @@ and fmt_let c ~ext ~rec_flag ~bindings ~parens ~fmt_atrs ~fmt_expr ~body_loc
   in
   let fmt_binding ~first ~last binding =
     let ext = if first then ext else None in
-    let in_ indent = fmt_if_k last (fmt_in indent) in
+    let in_ =
+      if last then
+        Some
+          { txt= (fun indent -> fmt_in indent)
+          ; loc=
+              (* I have checked the parser, this should never happen. *)
+              Option.value_exn
+                ~message:
+                  "Bug in the parser : let bindings with an should always \
+                   have a matching loc_in"
+                loc_in }
```

Comment:
This is not satisfying as `fmt_let` is not specific to an ast node and could be reused for other purposes.
Also, I doubt it as `pvbs_loc_in` is an option.